azure-iot-device/tests/iothub/pipeline/test_pipeline_stages_iothub_mqtt.py

```diff
@@ -131,6 +131,7 @@ def apply_fake_pipeline_thread(fake_pipeline_thread):
     handled_ops=ops_handled_by_this_stage,
     all_events=all_common_events + all_iothub_events,
     handled_events=events_handled_by_this_stage,
+    extra_initializer_defaults={"feature_to_topic": dict},
```

Comment:
Given that this dict maps features to topics, for use in other functions, and those tests test for correctness of topics chosen, I feel like this should probably be a private attribute that isn't tested. There's no real requirements value in proving that a mapping dictionary exists called `feature_to_topic`

Reply:
out of scope.  If you believe this is important, open a task in TFS.

---
In reply to: [312593187](https://github.com/Azure/azure-iot-sdk-python-preview/pull/160#discussion_r312593187) [](ancestors = 312593187)